pkg/models/deviceidfilter.go

```diff
@@ -4,27 +4,25 @@
 //
 // SPDX-License-Identifier: Apache-2.0
 
```

Comment:
Filename long ad difficult to read.

Reply:
This types are local to distro. It does not make any sense moving it to models.